### PR TITLE
Clean up inliner estimate code size

### PR DESCRIPTION
**Pull request by cathyzhyi**
Remove some dead code. Remove
setJ9EstimateCodeSizeMethod/getJ9EstimateCodeSizeMethod and replace it
with calltarget->_calleeMethod since they are the same.

Signed-off-by: Yi Zhang <yizhang@ca.ibm.com>

### CODE REVIEW

**Comment by 0xdaryl** [comment]
Minor request: could you fix the spelling of "inliner" in your commit title please?

**Comment by cathyzhyi** [comment]
> Minor request: could you fix the spelling of "inliner" in your commit title please?

fixed the spelling. thanks!

File: runtime/compiler/optimizer/J9EstimateCodeSize.cpp [hunk 1]
```diff
@@ -1161,17 +1043,12 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
...
                debugTrace(tracer(),"adding a fallthrough edge between block %p %d and %p %d", currentBlock, currentBlock->getNumber(), newBlock, newBlock->getNumber());
-
                debugTrace(tracer(),"joining nodes between blocks %p %d and %p %d", currentBlock, currentBlock->getNumber(), newBlock, newBlock->getNumber());
                currentBlock->getExit()->join(newBlock->getEntry());
-
                cfg.addEdge(currentBlock, newBlock, stackAlloc);
+               } 
+            else 
                addFallThruEdge = true;
-               }
```
Comment (andrewcraik) [inline runtime/compiler/optimizer/J9EstimateCodeSize.cpp]: can we keep the {}s for symmetry?

**Comment by andrewcraik** [comment]
Are the static methods you are removing just dead code?

File: runtime/compiler/optimizer/J9EstimateCodeSize.cpp (hunk 1 above)
Comment (cathyzhyi) [reply runtime/compiler/optimizer/J9EstimateCodeSize.cpp]: added back {} as suggested.

**Comment by cathyzhyi** [comment]
> Are the static methods you are removing just dead code?

yes, those methods are static methods of TR_PrexArgInfo.

**Comment by andrewcraik** [comment]
Jenkins test sanity xlinux,win,plinux jdk8,jdk11

**Comment by cathyzhyi** [comment]
the failures are unrelated to this change

**Comment by andrewcraik** [comment]
@cathyzhyi sorry I dropped the ball on this - could you rebase forward to re-run the tests? I think the change looks reasonable.

**Comment by andrewcraik** [comment]
Jenkins test sanity xlinux,win,plinux jdk8,jdk11

**Comment by andrewcraik** [comment]
Jenkins test sanity xlinux,win,plinux jdk8,jdk11

**Pull request merged**